.circleci/config.yml

```diff
@@ -108,11 +108,13 @@ workflows:
             branches:
               only: master
             tags:
-              only: /^v[0-9]+(\.[0-9]+)*$/
+              only: /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
```

Comment:
This has to be added at every step, otherwise it won’t trigger a build on tags (see https://github.com/zio/zio-sqs/blob/master/.circleci/config.yml)

Reply:
good point